packages/router/src/router_state.ts

```diff
@@ -386,6 +397,11 @@ export class ActivatedRouteSnapshot {
     return this._queryParamMap;
   }
 
+  /** The resolved route title */
+  get title(): string|undefined {
+    return this.data[TitleKey];
+  }
+
```

Comment:
```suggestion
  /** The resolved route title */
  readonly title = this.data[TitleKey];
```
(and move this line up above the constructor)